### ISSUE TITLE
[1.16.x 4.2.0] Ores from SimpleOres not generating 

### ISSUE THREAD

**Issue by Calendis** [oitsjustjose/Geolosys#251]
I have added Mythril, Adamantium, and Onyx to geolosys.json, but they do not seem to generate veins.

I inserted this into the "ores" list

`
{
      "name": "Mythril",
      "type": "DENSE",
      "oreBlocks": [
        "simpleores:mythril_ore",
        100
      ],
      "maxY": 23,
      "minY": 0,
      "chance": 100,
      "size": 400,
      "density": 0.8,
      "dimBlacklist": [
        "the_nether",
        "the_end"
      ]
    }
`

In the above example, I have cranked up the chance and size just to see if I could find any, and I cannot.

**Comment by oitsjustjose** [comment]
You're missing an entry for `sampleBlocks`, that's probably it.

**Comment by Calendis** [comment]
I added a sampleBlocks entry `"sampleBlocks": [
        "simpleores:mythril_ore",
        100
      ]`, but I still couldn't find any deposits.

I also noticed some strange inconsistencies... the configs generated by the configurator use the keys "minY" and "maxY", but the config generated by the mod uses "yMin" and "yMax". Also my config uses "samples" instead of "sampleBlocks". What is the difference? When I try to use "samples" rather than "sampleBlocks" in my mythril_ore entry my game crashes even though the chances certainly sum to 100. I can provide the crash log if needed. I also tried changing "minY" and "maxY" to "yMin" and "yMax", seemingly to no effect.

Edit:
Finally got it working, sorry for the potentially spurious issue report. I manually created the entries rather than trying to paste from the configurator's output, and it works great! I don't know if the differences between the names of the keys was the problem, or if I had made some subtle syntax error during pasting. Thanks for the reply
